src/GridView/GridView.js

```diff
@@ -1,32 +1,54 @@
-import React, { Component } from 'react';
-import DonutChart from './DonutChart'
-import HierarchyStepper from './HierarchyStepper'
-import SearchBar from './../Utils/SearchBar'
+import React, {Component} from 'react';
+import {connect, PromiseState} from 'react-refetch';
+import LoadingAnimation from './../Utils/LoadingAnimation';
+import ErrorMessage from './../Utils/ErrorMessage';
+import DonutChart from './DonutChart';
+import HierarchyStepper from './HierarchyStepper';
+import SearchBar from './../Utils/SearchBar';
 import CardGrid from './CardGrid';
 
-class GridView extends Component {
-
-
-	componentDidMount() {
-		this.props.datasource.getEntities(3, this.getEntities);
-	}
 
-	getEntities(entities) {
-		this.setState({
-			entities: entities
+class GridView extends Component {
+	
+	getChildEntityTypes(parentEntityTypeId, hierarchy) {
```

Comment:
I do see your point concerning the "child" names in CardGrid. But in GridView we should stick to this name, to make clear that we are talking about the types of child elements.